tests/framework/installer/ceph_manifests_v0.9.go

```diff
@@ -215,7 +215,7 @@ rules:
   - update
   - delete
 - apiGroups:
-  - extensions
+  - apps
```

Comment:
For the upgrade test, we'll need to revert any changes to this file

Reply:
I have a change coming in #2522 (hopefully merged today) that will update the rbac needed for the upgrade test. We'll need to do something similar here. This helps us catch what needs to be upgraded by users for the next release.

Reply:
done